index.js

```diff
@@ -13,12 +13,17 @@ module.exports.process = (output, origin) => {
     subtitleUrl = `${origin}/ytdl/vtt?suburi=${encodeURIComponent(subtitleFile.subs.url)}`;
   }
 
+  const { url } = data;
```

Comment:
should this part be immediately after the JSON.parse?

Reply:
Sure